docs/.vitepress/vitepress/components/vp-app.vue

```diff
@@ -55,6 +55,7 @@ onMounted(async () => {
   )
 
   if (lang.value === 'zh-CN') {
+    if (location.href.startsWith('https://element-plus.gitee.io')) return
```

Comment:
Suggestion: we can use location.host to judge

Reply:
ok.

Reply:
Right, this was my big mistake sorry :bow: